executor_internal_test.go

```diff
@@ -0,0 +1,290 @@
+package taskrunner
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/samsarahq/taskrunner/config"
+	"github.com/samsarahq/taskrunner/shell"
+	"github.com/stretchr/testify/assert"
+)
+
+func BoolPtr(i bool) *bool {
```

Comment:
being really nitty but no need to make these uppercase (which exports the function) - `BoolPtr` -> `boolPtr`

I got a bit confused why we were exporting these functions, then I realized this is in a test file anyways so it doesn't really get exported to users, but confused me for a bit